ADRs/RH/SIG-SRE/ADR-00002 Personas related to Managed Services.md

```diff
@@ -0,0 +1,119 @@
+# ADR-00002 Personas related to Managed Services
+
+Authors: Craig Robinson
+
+
+## Status
+
+Draft
+
+
+## Problem Statement
+
+There is content within SIG-SRE that is being produced that depends upon the defintion of a "persona" as it is related to Managed Services. For example, as a consumer of SLO's, I want to be able to consume the data in a relevant and consistent way in accordance with my "persona". Similarly, there is a RACI chart which has persona as a key datapoint.
+
+Therefore, to assist in the production of this other content, it is necessary to first define the persona. Some examples are (but not limited to): SRE, Engineer, Product Owner, Business, Executive.
```

Comment:
nit: `Business, Executive` isn't listed below.  Probably intended to match `Exec (VP+)`.  But that seems very specific for "here's the type of thing we mean by persona", so maybe just replace with `etc.`?

Reply:
@wking This sentence has now been removed, so the question is null now. I will resolve this one.